src/types.js

```diff
@@ -13,7 +13,7 @@ export type MapStateToProps = any; // { MapStateToProps } from 'react-redux';
 export type * from './actionTypes';
 export type * from './api/apiTypes';
 
-export type Thunk<A> = ((Dispatch, GetState) => Promise<void> | void) => A;
+export type Thunk<A> = ((Dispatch, GetState) => Promise<any> | void) => A;
```

Comment:
I think we should keep it `void` and not return values from these actions?

Just adding `{` and `}` around toggleStreamNotification's body and turning it to `Promise<void>` seems to me to match our intent better.

Reply:
Yeah, I tried that initially. However, toggleStreamNotification is an `async` function, so it'll always return a promise. And that promise will not have the type void.

Reply:
It will if you add the `{` and `}` around the function. Right now it is returning the result of `apiPost` and we are not using that.

Reply:
Aah, good point! I looked at toggleStreamNotification*s*, didn't think about toggleStreamNotification